docs/dev/api-reference/xml-rpc.rst

```diff
@@ -20,30 +20,29 @@ Example usage (Python 3)::
   >>> import pprint
   >>> import time
   >>> client = xmlrpc.client.ServerProxy('https://pypi.org/pypi')
-  >>> client.package_releases('roundup')
-  ['1.6.0']
+  >>> client.changelog_last_serial()
+  24891357
   >>> time.sleep(1)  # Sleep to avoid rate limit
-  >>> pprint.pprint(client.release_urls('roundup', '1.6.0'))
-  [{'comment_text': '',
-  'digests': {'md5': '54d587da7c3d9c83f13d04674cacdc2a',
-              'sha256': '1814c74b40c4a6287e0a97b810f6adc6a3312168201eaa0badd1dd8c216b1bcb'},
-  'downloads': -1,
-  'filename': 'roundup-1.6.0.tar.gz',
-  'has_sig': True,
-  'md5_digest': '54d587da7c3d9c83f13d04674cacdc2a',
-  'packagetype': 'sdist',
-  'path': 'f0/07/6f4e2164ed82dfff873ee55181f782926bcb4a29f6a83fe4f8b9cbf5489c/roundup-1.6.0.tar.gz',
-  'python_version': 'source',
-  'sha256_digest': '1814c74b40c4a6287e0a97b810f6adc6a3312168201eaa0badd1dd8c216b1bcb',
-  'size': 2893499,
-  'upload_time_iso_8601': '2018-07-13T11:30:36.405653Z',
-  'url': 'https://files.pythonhosted.org/packages/f0/07/6f4e2164ed82dfff873ee55181f782926bcb4a29f6a83fe4f8b9cbf5489c/roundup-1.6.0.tar.gz'}]
+  >>> client.changelog_since_serial(24891357)
+  [['py-pcapplusplus', '1.0.0', 1725534675, 'remove release', 24891358]]
+  >>> time.sleep(1)  # Sleep to avoid rate limit
+  >>> pprint.pprint(client.list_packages_with_serial())
+  {'0': 3075854,
+   '0-._.-._.-._.-._.-._.-._.-0': 1448421,
+   '0-core-client': 3242044,
+   '0-orchestrator': 3242047,
+   '0.0.1': 3430659,
+   '0.618': 14863648,
+  ...
 
 .. _changes-to-legacy-api:
 
 Changes to XMLRPC API
 ---------------------
 
+- ``list_packages``, ``package_releases``, ``release_urls``, and ``release_data``
+  permanently deprecated and disabled. See `Deprecated Methods`_ for alternatives.
```

Comment:
Yay! Should the existing changes entires remain for `package_releases` and `release_data` a few lines below this, or should they get strikethrough/removed?

Reply:
Good catch! 044226077